Security/Standard_Levels.mediawiki

```diff
@@ -0,0 +1,311 @@
+* '''Effort''': ''A larger amount of effort'' is expected.
+* '''Risk acceptance''': Risk must  be discussed, and should at least be mitigated.
+* '''Exception time''': Recommend remediation within 30-90 days.
+|-
+! <span style="padding: 0 32px;background-color:#E77471;">MAXIMUM</span>
+|
+* Red signifies "most important".
+* Maximum is a level. Critical is not.
+| ''This is the most important level, where the risk is especially great.''
+
+* '''Attention''': Get full attention from all concerned parties
+* '''Complexity''': Following practices with security level maximum is expected. Uses the best and most advanced available remediation or mitigation strategies, tools.
+* '''Effort''': Greatest amount of effort is required to mitigate or remediate the risk.
+* '''Risk acceptance''': This risk is rarely expected to be accepted as residual risk, must be discussed, and should be mitigated or remediated.
+* '''Exception time''': Recommend fixing immediately.
+|-
+
+== Effort measurements ==
+When work is required for a remediation, mitigation, additional security control, etc. we rate this effort using our standard levels. This helps managing resources and prioritizing work.
+
+{| class="wikitable"
+|-
+! Level
+! Time spent
+|-
+! <span style="padding: 0 32px;background-color:lightgrey;">LOW</span>
+|
+* Small group: Few hours.
+* Large group: Few minutes.
+
+Example: Flip a configuration switch, change a password, lookup a document, etc.
+|-
+! <span style="padding: 0 32px;background-color:steelblue;">MEDIUM</span>
+|
+* Small group: A few days.
+* Large group: A few hours.
+
+Example: Take a group decision, create standards, lookup complex data, manual upgrades, etc.
+|-
+! <span style="padding: 0 32px;background-color:#FFF380;">HIGH</span>
+|
+* Small group: A week or two.
+* Large group: A day or two.
+
+Example: Implement a new security control, code a new feature, change all company user passwords, etc.
+|-
+! <span style="padding: 0 32px;background-color:#E77471;">MAXIMUM</span>
+|
+* Small group: Weeks, quarterly goals.
+* Large group: Weeks, quarterly goals.
+
+Example: Implement a new security design/change product design, etc.
+|-
+
+== RFC2119 handling recommendation levels ==
+{| class="wikitable"
+|-
+! Level
+! Coding rationale
+! Definition & expectations
+|-
+|-
+! <span style="padding: 0 32px;background-color:lightgrey;">OPTIONAL</span>
+|
+* See risk level's "low" rationale.
+|
+* This is up to the reader to choose to follow or not to follow this recommendation.
+|-
+! <span style="padding: 0 32px;background-color:#FFF380;">SHOULD</span>
+|
+* See risk level's "high" rationale.
+|
+* Should is very close to "must". It means something should be done unless there is a strong reason not to. Thus it's highly important.
+|-
+! <span style="padding: 0 32px;background-color:#E77471;">MUST</span>
+|
+* See risk level's "maximum" rationale.
+|
+* This must be done - there is no exception.
```

Comment:
What I mean by colloquial is that I would probably say something like "Mandatory, without exception."